lib/util/constants.ts

```diff
@@ -52,6 +52,7 @@ export const ErrorCodes = {
   },
   ResponseValidationError: { name: "RESPONSE_VALIDATION_ERROR", id: "OAV108" },
   RequestValidationError: { name: "REQUEST_VALIDATION_ERROR", id: "OAV109" },
+  RoundtripValidationError: { name: "ROUNDTRIP_VALIDATION_ERROR", id: "OAV135" },
```

Comment:
Please add this error code to errors definition document in a separate pr.

Reply:
Sure, may I ask where should I add the definition.